.pre-commit-config.yaml

```diff
@@ -884,7 +884,6 @@ repos:
       - id: compile-fab-assets
         name: Compile FAB provider assets
         language: node
-        'types_or': [javascript, ts, tsx]
```

Comment:
This is why sometimes the assets are not generated. If you modify `package.json` which define the dependencies, it wont regenerate the assets, although it should. Removing this filter fixes the issue